src/dataframe/dataframe.jl

```diff
@@ -1206,7 +1206,7 @@ end
 
 Change columns selected by `cname` or `cnames` in data frame `df`
 to `CategoricalVector`. If no columns are indicated then all columns that have
-an `AbstractString` element type type will be converted to categorical.
+an `Union{AbstractString, Missing}` element type type will be converted to categorical.
```

Comment:
To avoid any ambiguity, I'd say "all columns whose element type is a subtype of `Union{AbstractString, Missing}` will be...".

Reply:
fixed